lib/project.js

```diff
@@ -1,9 +1,12 @@
+const ProjectLinks = require('./projectLinks');
+
 class Project{
   constructor(channel){
     this.channel_id = channel;
+    this.links = new ProjectLinks();
```

Comment:
You'll want to pass along the `channel` and `dbDriver` variables to this `ProjectLinks` class.